docs/backstage/catalog-info.yaml

```diff
@@ -45,7 +45,7 @@ metadata:
   namespace: default
 spec:
   memberOf:
-    - engineering
+    - [engineering]
```

Comment:
was this intentional?

Reply:
it was, as this is an array field. but i agree with your comment below that we shouldn't be too strict in this case, so i'll revert this when that's fixed